Scripts/Player_Scripts/player.gd

```diff
@@ -0,0 +1,37 @@
+extends CharacterBody3D
+
+
+@export var SPEED = 5.0
+@export var JUMP_VELOCITY = 4.5
+var mouse_sensitivity = 0.002
+
+func _input(event):
+	if event is InputEventMouseMotion and Input.mouse_mode == Input.MOUSE_MODE_CAPTURED:
+		rotate_y(-event.relative.x * mouse_sensitivity)
+		$Camera3D.rotate_x(-event.relative.y * mouse_sensitivity)
+		$Camera3D.rotation.x = clampf($Camera3D.rotation.x, -deg_to_rad(70), deg_to_rad(70))
+		
+func _ready():
+	Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
+
+func _physics_process(delta: float) -> void:
```

Comment:
Physics process has three distinct functions, "ApplyGravity", "HandleJump" and "Handle Movement"

please abstract these into independent functions.